samples/nrf9160/http_update/full_modem_update/README.rst

```diff
@@ -13,7 +13,7 @@ The sample downloads a modem firmware signed by Nordic and performs the firmware
 Overview
 ********
 
-An external flash memory with a minimum of 2MB of free space is required to perform a full modem update.
+An external flash memory with a minimum of 4MB of free space is required to perform a full modem update.
```

Comment:
I don't like the 4Mb requirement to be listed several places. 
Would it be possible to only keep the size requirement here (in this file) and link to this from the two other location?

Reply:
fixed